.github/workflows/ci.yml

```diff
@@ -71,6 +71,11 @@ jobs:
       run: |
         pytest --random-order --cov=freqtrade --cov=freqtrade_client --cov-config=.coveragerc
 
+    - name: Run Pester tests with code coverage
```

Comment:
I think this should happen in the windows section of CI ... (or get it's own job in the flow).

i doubt that the GHA environment for linux has powershell installed - and if it does, then i doubt it'll be suitable for the tests we're tryng to run.

Reply:
https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-powershell

i tried to do it like there.